delivery_schenker_picking_volume/models/delivery_carrier.py

```diff
@@ -6,7 +6,5 @@
 class DeliveryCarrier(models.Model):
     _inherit = "delivery.carrier"
 
-    def _prepare_schenker_shipping(self, picking):
-        vals = super()._prepare_schenker_shipping(picking)
-        vals["shippingInformation"]["volume"] = picking.volume
-        return vals
+    def _schenker_shipping_information_product_volume(self, product, qty):
+        return product._get_volume_for_qty(qty)
```

Comment:
This depends on https://github.com/OCA/stock-logistics-warehouse/pull/1790 so it should fail as it is not merged yet. Can you add a unittest for this?

Reply:
I added this PR to the test-requirements.txt so it works, but yes there should be a test for it.

Reply:
There is now a unittest for it